api/src/main/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceApplication.java

```diff
@@ -47,6 +57,20 @@ public Clock clock() {
         return Clock.systemDefaultZone();
     }
 
+    @Bean
+    public ObjectMapper objectMapper() {
```

Comment:
I'm personally against having to define these things (convention over configuration and all that), but if we must do this, could we add this to the java-common package instead so that we don't have to do this kind of boiler plate code in all our projects. I'd be interested to know what failed and why these date formats are not standard anymore too.

Reply:
We don't have to - this was my mistake.